src/app.js

```diff
@@ -217,9 +217,17 @@ function run () {
         var number = document.querySelector('#value').value
         var select = document.getElementById('unit')
         var index = select.selectedIndex
-        var selectedUnit = select.querySelectorAll('option')[index].innerHTML
-        var unit = ''
-        if (selectedUnit === 'wei') { unit = 'wei' }
+        var selectedUnit = select.querySelectorAll('option')[index].dataset.unit
+        var unit = 'ether' // default
+        if (selectedUnit === 'ether') {
+          unit = 'ether'
+        } else if (selectedUnit === 'microether') {
+          unit = 'microether'
```

Comment:
Is anyone using this? I'd argue not the only ones used are `ether`, `gwei`, `wei` and at some point in the past `finney` was also used.

Reply:
+1 finney

Reply:
So this is what users are really operating with? I was coming from the side of a new user, something that I can wrap my head around.

But tell me what are units you guys rotate in your heads?

Reply:
I will go with wei, gwei, finney and ethere for now then!

Reply:
Actually, should we add `szabo` (nobody uses it) only because Solidity supports it? Downside: if it is added people might feel encourage.